specificatie/openapi.yaml

```diff
@@ -648,6 +810,132 @@ components:
+          $ref: '#/components/schemas/Geometrie'
+        spatialOperator:
+          type: string
+          description: >-
+            "Een operator uit de set van spatial operators die gedefinieerd zijn in ISO 19125-1. Ondersteund worden:
+            - intersects  : object geometrie `overlapt` of `raakt` de opgegeven contour
+            - within      : object geometrie valt volledig binnen de opgegeven contour
+            - contains    : object geometrie omsluit de opgegeven contour volledig"
+          enum:
+            - intersects
+            - within
+            - contains
+          default: intersects
+        tolerantie:
+          description: 'De maximale afwijking (in meters) tussen twee punten waarbij deze punten nog als één punt worden gezien.'
+          type: number
+          format: float
+          default: 0.001
+          minimum: 0.00001
+          maximum: 1
+
+    Geometrie:
+      type: object
+      description: 'De geometrie waarmee gezocht moet worden naar objecten. Er kan een punt, cirkel of polygoon worden opgegeven.' 
+      properties:
+        locatie:
+          $ref: 'http://schemas.opengis.net/ogcapi/features/part1/1.0/openapi/schemas/pointGeoJSON.yaml'
+        cirkel:
+          $ref: '#/components/schemas/Cirkel'
+        polygoon:
+          $ref: 'http://schemas.opengis.net/ogcapi/features/part1/1.0/openapi/schemas/polygonGeoJSON.yaml'
+        
+    Cirkel:
+      allOf: 
+        - $ref: 'http://schemas.opengis.net/ogcapi/features/part1/1.0/openapi/schemas/pointGeoJSON.yaml'
+        - type: object
+          description: 'Bij een geometrie type cirkel kan naast een punt een straal (eenheid meters) worden meegegeven, zodat in een gebied met een streal rondom dat punt wordt gezocht.'
+          properties:
+            radius:
+              type: number
+              default: 100
+              minimum: 1
+              maximum: 280
+
+    Waardebereik:
+      description: >-
+        Een generieke aanduiding voor de min en max waarde van een attribuut van een object.
+        Als alleen een min waarde wordt opgegeven, dan wordt er gezocht naar waarden die groter dan of gelijk zijn aan de min waarde.
+        Als alleen een max waarde wordt opgegeven, dan wordt er gezocht naar waarden die kleiner dan of gelijk zijn aan de max waarde.
+        Als zowel min en max waarde worden opgegeven waarbij min < max dan wordt er gezocht naar waarden die groter dan of gelijk zijn aan min en kleiner dan of gelijk aan max, oftewel waarden die tussen min en max liggen.
+        Als zowel min en max waarde worden opgegeven waarbij min > max dan wordt er gezocht naar waarden die groter dan of gelijk zijn aan min en kleiner dan of gelijk aan max, oftewel waarden die niet tussen min en max liggen. 
+      type: object
+      properties:
+        min:
+          type: number
+        max:
+          type: number
+
+    LijstOperator:
+      type: string
+      enum:
+        - and   # alle waarden
```

Comment:
kan je de comments met betekenis van de operators in de description opnemen? Dan ziet iemand deze ook in Swagger of Redoc